### DISCUSSION TITLE
Single liner custom lookup

### DISCUSSION THREAD

**Discussion by agneevX** [AdguardTeam/AdGuardHome#3228] [Q&A] [answered]
I want to redirect all queries for `www.routerlogin.net www.routerlogin.com routerlogin.net routerlogin.com` to 10.0.0.1.

Can a single one liner be created to accomplish this?

Thanks!

I've tried `||routerlogin.*^$dnsrewrite=NOERROR;A;10.0.0.1` but that redirects every TLD for that domain (as it should).

EDIT: ABP style `10.0.0.1 www.routerlogin.net www.routerlogin.com routerlogin.net routerlogin.com` works but I would like something smaller if that's possible.

**Comment by ameshkov** [comment, marked as answer]
Well, you can do regular expressions:
`/(www.routerlogin.net)|(www.routerlogin.com)|(routerlogin.net)|(routerlogin.com)/$dnsrewrite=10.0.0.1`

or this:
`/(www.)?routerlogin.(net|com)/$dnsrewrite=10.0.0.1`
